lib/mayaUsd/render/vp2RenderDelegate/proxyRenderDelegate.cpp

```diff
@@ -1108,6 +1108,9 @@ bool ProxyRenderDelegate::getInstancedSelectionPath(
         return false;
     }
 
+    if (_proxyShapeData->ProxyShape())
```

Comment:
Is this going to catch only marquee selections in the viewport?  What about selections made in the Outliner, or any other future UFE-enabled node viewer or editor?  How would they receive these pre selection and post selection changed notifications?  Is there no way to reformulate this as standard UFE selection changed notifications?  It feels awkward to go through the proxy shape to distribute the notifications, when the UFE global selection itself is already observable, and is not tied to viewport-based selection.

Reply:
We're looking into removing (or at least moving somewhere else) these notifications.